utils/env_preferences/src/parse/posix.rs

```diff
@@ -0,0 +1,621 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+use icu_locale::extensions::unicode::{key, value};
+use icu_locale::extensions::Extensions;
+use icu_locale::subtags::{language, script, variant, Language, Region, Variants};
+use icu_locale::{LanguageIdentifier, Locale};
+
+// An incomplete list of POSIX language names that need to be converted to a BCP-47 equivalent
+// Some of these are approximate and not exact (e.g. "C"->"und")
+// This is based on real-world data only, so will hopefully cover most of the more common edge cases
```

Comment:
`nit`: please, provide source of the keys for this list. In the references discussion you list a gist, which is documenting a single generation from arch distro. Let's peel this onion. Where does that program take the data from? Who maintains the database that is used to generate it?
Please, find that and link it here.